src/worker/worker.c

```diff
@@ -270,7 +270,6 @@ static void kill_job(struct nm_event_execution_properties *event)
 	 */
 	if (cp->ei->state != ESTALE) {
 		timeouts++;
```

Comment:
`id` and `pid` are no longer used. Either remove the assignments or refactor to use them where we use `cp->ei->pid`. Which ever is fine by me.